### ISSUE TITLE
[BUG] Unexpected behavior of v-icon with dynamic icons

### ISSUE THREAD

**Issue by iranagame** [mathe42/vuetify-icon-injector#21]
Some dynamic icons including v-rating, dynamic v-icon (if passed dynamically via template) are missing and not printing, also v-text prop on v-icon throw this type error:
`TypeError: Cannot read property 'text' of undefined`

**Comment by mathe42** [comment]
As you see here: https://github.com/EC-Nordbund/vuetify-icon-injector/blob/dev/src/vuetifyIconProps.ts#L56 v-rating is in the list.

For v-icon I need a code snipped how you used v-text.

**Comment by iranagame** [comment]
For dynamic v-icon i do the followings:
`<v-icon>{{ item.icon }}</v-icon>` (Print nothing)
`<v-icon v-text="item.icon">{{ item.icon }}</v-icon>` (Print nothing)
`<v-icon v-text="item.icon"></v-icon>` (Throw that TypeError)

**Comment by mathe42** [comment]
Ah!

Thats not a bug...

The full `mdi-...` string has to be IN the template and not in the script section. In your script section you have to import the icons from `@mdi/js`. For simple if statements you can do:

```html
<v-icon>{{ usertype == 'admin' ? 'mdi-icon1' : 'mdi-icon2' }}</v-icon>
<v-rating :empty-icon="usertype == 'admin' ? 'mdi-icon1' : 'mdi-icon2'"></v-rating>
```

There is no easy way to look at the script section and replace that strings - that is also a thing I wouldn't recomend it could break VERY easy.

**Comment by iranagame** [comment]
Try this please, it's print nothing!
`<v-icon>{{ 'mdi-card-text-outline' }}</v-icon>`

**Comment by mathe42** [comment]
Look at this in the weekend. Should work... Can you provide a full reproduction?

**Comment by mathe42** [comment]
You might be right. In my main project I don't use v-icon with dynamic content. I will have a deeper look.

**Comment by mathe42** [comment]
Does `<v-icon v-text= "'mdi-card-text-outline'" />` work?

**Comment by iranagame** [comment]
No, it will throw a TypeError!

**Comment by mathe42** [comment]
I changed a few month ago the packagename to @ec-nordbund/vuetify-icon-injector didn't push the update of the readme. Did you installed vuetify-icon-injector or @ec-nordbund/vuetify-icon-injector. With @ec-nordbund/vuetify-icon-injectorthe following works for me (for v-html and v-text see the single quotes in the double quotes):

```vue
    <v-icon v-html="'mdi-card-text-outline'"/>
    <v-icon v-text="'mdi-card-text-outline'"/>
    <v-icon v-html="'mdi-card-text-outline'"></v-icon>
    <v-icon v-text="'mdi-card-text-outline'"></v-icon>
    <v-icon>{{ 'mdi-card-text-outline' }}</v-icon>
```

Make shure you use the latest version (2.3.2). [EDIT: Just released 2.3.3 and 2.3.4 with the updated readme]

Hope this helped.

**Comment by mathe42** [comment]
I close this for now. If it is not fixed for you please share a reproduction so I can see what is going wrong and debug it.
